app/controllers/organizations_controller.rb

```diff
@@ -19,6 +19,9 @@ def new
 
   def create
     @organization = Organization.new(new_organization_params)
+    if Classroom.flipper[:explicit_assignment_submission].enabled? current_user
+      @organization.setup_webhook(webhook_events_url)
```

Comment:
@tarebyte I was doing this in the `before_validation(on: :create)` callback of the `Organization` model. However, as we must have the `webhook_events_url` in order to setup the webhook, including `Rails.application.routes.url_helpers` in `Organization` and setting the `host` of the application in `environment/*.rb` are necessary.

Is it fine to place this piece of code here or we have better options?